docs/how-tos/sensor-module.md

```diff
@@ -29,12 +29,29 @@ Making a module to support your sensor will allow you to use it with Viam's data
 1. [Start with a test script](#start-with-a-test-script)
 1. [Generate template module code](#generate-template-module-code)
 1. [Implement the sensor API](#implement-the-sensor-api)
-1. [Make the module executable](#make-the-module-executable)
 1. [Test your module locally](#test-your-module-locally)
 1. [Upload your module](#upload-your-module-to-the-registry)
 
 {{% /alert %}}
 
+## Prerequisites
+
+{{< expand "Install the Viam CLI and authenticate" >}}
+Install the Viam CLI and authenticate to Viam, from the same machine that you intend to upload your module from.
```

Comment:
Authenticate is the next bit here, so probably cleaner to put them one after the other
```suggestion
Install the Viam CLI on the same machine that you intend to upload your module from.
```